### PR TITLE
Adds WAY more Class-D freedom, making people become more interested upon roundstart join, improves the EZ Brig area, changes spelling mistakes

### PR DESCRIPTION
**Pull request by pumpkintheproot**
## About the Pull Request

What this pull request does is essentially causes a lot more people to be interested in the game when roundstart joining as a Class-D (AKA the first role you are presented with) because quite frankly, waiting 20 minutes for a guard to manually open the cells, and another 10 for job assigning gets quite boring and makes people, even ones that have played for a while, want to quit due to the lack of action. This pull request makes it so **cell doors are open on roundstart (hot change)**, Auto Assignment machines are unlocked on roundstart (since guards dont know how to unlock it ((sometimes))) and medical shutters automatically open so that any Class-D can get medical attention themselves without guard intervention (still need a medcard)

The second part of this PR is that it improves the EZ brig area. As of now, the EZ brig cells require access to open, and prisoners do not have that, so I changed it so that there is no access requirements, letting prisoners exit and enter as they please. I also added flashes to the cells and bolts to the cell doors so you can bolt any prisoners inside of the cells incase of trouble.

Also fixes spelling mistakes such as "outter" and "checkkpoint" (mentioned in #1482)

## Why It's Good For The Game
Gameplay gets boring as a D-Class, especially waiting 20 minutes in a locked cell with no way out and a guard having to manually press a button.

In the EZ Brig cell, it gets boring just sitting in a room doing nothing, adding cells avaliability improves this by atleast having somewhere to sleep

## Changelog

:cl:
qol: Made it so blast doors on Class-D cell doors are automatically opened on roundstart, reducing the chance of leaving due to lack of gameplay
change: Auto Assignment Machines are now unlocked on roundstart (probably temporarily) as it does not have any codex/information on how to unlock it, and guards may not know how to do this
change: Med shutters are now open roundstart, making it so D-Class can go to medical with a med-card and get medical attention themselves without a guard needing to open the shutters
qol: added no access requirement to ez brig cells, meaning prisoners can go in these cells without a guard
change: added flashes and bolts to cell doors, allowing you to put any prisoners in there, similar to isolation in CDCZ
change: spelling of certain buttons and areas, improving the readability of some areas and buttons
:cl:

### CODE REVIEW

**Review by pumpkintheproot — COMMENTED** [review]
deleted cuz this pr aint about this site level 👎

**Comment by pumpkintheproot** [comment]
this issue resolves #1482

**Comment by pumpkintheproot** [comment]
windows broken because i goofed with replace, pls wait while i fix this goof 😃

**Comment by pumpkintheproot** [comment]
goof fixed 👍

**Comment by cheesePizza2** [comment]
This sounds like a really bad idea, there's a reason D-class aren't let out until there's a guard able to watch them

also don't be anonhault, atomize your PR

**Comment by pumpkintheproot** [comment]
> This sounds like a really bad idea, there's a reason D-class aren't let out until there's a guard able to watch them
> 
> also don't be anonhault, atomize your PR

jokes on you i dont know what atomizing pr's mean 😎 
also, with lowpop and all, and the boundaries of the server rules, since you cant do anything until 30 minutes have passed, theres no danger to keeping d-class out as a guard will (most) likely be there by 30 minutes. its just a way to reduce the boredomness of waiting as ive seen lots of people resort to using OOC or even admin intervention because they cant do anything since there are no guards

**Review by EgorDinamit — APPROVED** [review]
Sounds fun. Let's give it a merge.

**Comment by EgorDinamit** [comment]
Wow, failing checks :)

File: maps/site53/site53-1.dmm [hunk 1]
```diff
@@ -14411,7 +14439,10 @@
 	req_access = list(list("ACCESS_MEDICAL_LEVEL2","ACCESS_SECURITY_LEVEL2"))
 	},
 /obj/machinery/door/blast/shutters/open{
-	begins_closed = 1;
+	id_tag = "CDZMedbaywin";
+	name = "CDZ Medbay Window Shutters"
+	},
+/obj/machinery/door/blast/shutters/open{
```
Comment (aidan-bowman) [inline maps/site53/site53-1.dmm:14445]: there are two shutters here?

File: maps/site53/site53-1.dmm [hunk 2]
```diff
@@ -16023,7 +16054,9 @@
 /turf/simulated/floor/tiled/white,
 /area/site53/uhcz/scp8containment)
 "Mm" = (
-/obj/machinery/job_terminal,
+/obj/machinery/job_terminal{
+	locked = 0
```
Comment (aidan-bowman) [inline maps/site53/site53-1.dmm:16058]: does this even matter? they can't be given jobs until a guard mans the post anyways

Comment (pumpkintheproot) [reply maps/site53/site53-1.dmm:16058]: yes.. a guard has to manually unlock the auto assignment machines

File: maps/site53/site53-1.dmm (hunk 1 above)
Comment (pumpkintheproot) [reply maps/site53/site53-1.dmm:14445]: yes

**Comment by cheesePizza2** [comment]
jesus christ dude don't mark what i post as resolved unless it's ACTUALLY RESOLVED

**Comment by pumpkintheproot** [comment]
stop making useless comments then you donk pocket!!

**Comment by cheesePizza2** [comment]
Fixes #1482 btw

**Pull request merged**